manifests/params.pp

```diff
@@ -78,6 +79,7 @@
           default  => pick($datadir, "/var/lib/pgsql/${version}/data"),
         }
         $confdir                = pick($confdir, $datadir)
+        $postgresql_conf_mode   = pick($postgresql_conf_mode, '0600')
```

Comment:
Could you please clarify on  different permission modes on Amazon and Debian. Thank you

Reply:
I don't know what Amazon should be since I have no access to it, but on a fresh CentOS 7:
```
-rw-------. 1 postgres postgres 19845 Nov 16 16:10 /var/lib/pgsql/data/postgresql.conf
```

Then on a fresh Debian 10:
```
-rw-r--r-- 1 postgres postgres 24160 Nov 16 16:37 /etc/postgresql/11/main/postgresql.conf
```

I'm not sure how many versions I checked when I originally wrote the change, but I recall that Debian and Ubuntu versions were the same.